1_turb_abl_fsi/turb_run/nrel5mw_nalu.yaml

```diff
@@ -178,15 +189,14 @@ realms:
         angle: 5.0
         axis: [0.0, 1.0, 0.0]
         centroid: [0.0, 0.0, 0.0]
+      - type: translation
+        displacement: [1772.54, 2080.0, 90.0]
         
-  - name: turbine_translate
-    mesh_parts:
-      - blade1-HEX
-      - blade2-HEX
-      - blade3-HEX
+  - name: tower_translate
+      - tower-fluid-HEX
     motion:
       - type: translation
-        displacement: [1772.54, 2080.0, 90.0]
+        displacement: 1800.0, 1800.0, 90.0]
```

Comment:
I think this should just be rotor position ([1772.54, 2080.0, 90.0]) + (5, 0, 0). This also needs to be consistent with inp.yaml